### DISCUSSION TITLE
How to compile hello_world in windows OS

### DISCUSSION THREAD

**Discussion by achzhou** [hosseinmoein/DataFrame#263] [Q&A]
I got all the DataFame source codes from github. I would like to run hello_world as a start. But I don't know how to do that in windows. I am not quite familiar with cmake. I don't have make. I have cmake, nmake from windows, cl.exe as C++ compiler. Follow the readme.md I have the library compiled. But I am not sure if I could get the library working so I need to try to compile the hello_world. 

In windows, from windows command prompt, how am I supposed to do it? how to use cmake then what?

Thanks!

**Comment by hosseinmoein** [comment]
I am also not familiar with Windows development environment. But if you have followed the readme instructions and already compiled the library with cmake, then hello_world and all test executables should be already made and ready to execute. 

DataFrame is also available on VCPKG. You may want to look at that too
